<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnDialog</class>
 <widget class="QDialog" name="LearnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>设备类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>遥控器</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>探测器</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="learnPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>对码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deltePushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>分地址</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="explainLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
